server/src/main/java/org/eclipse/hono/server/HonoServer.java

```diff
@@ -322,21 +323,22 @@ protected int getInsecurePortDefaultValue() {
     }
 
     /**
-     * Gets the insecure port Hono listens on for AMQP 1.0 connections (if configured).
+     * Gets the insecure port Hono is determined to listen on for AMQP 1.0 connections (if configured).
      * <p>
      * If the port has been set to 0 Hono will bind to an arbitrary free port chosen by the operating system during
      * startup. Once Hono is up and running this method returns the <em>actual port</em> Hono has bound to.
      * </p>
      *
      * @return the insecure port Hono listens on.
      */
-    public int getActualInsecurePort() {
+    @Override
+    public int getInsecurePort() {
```

Comment:
why does this method contain a check for `config.isInsecurePortUnconfigured() && !config.isInsecurePortEnabled()` while the `getPort()` method does not contain such a check?